EIPS/eip-4902.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: 4902
+title: Decentralized Autonomous Access (DAA)
+description: DAAs represent an access over digital or physical things & resources, with the ability to connect with entities that seeking to use it
+author: <Daniel @eartho-group>
+discussions-to: https://github.com/ethereum/eips/issues/4902
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-22
+requires: 20
+---
+
+## Abstract
```

Comment:
Abstract should be relatively short, maybe 1 paragraph.

Reply:
its really important to me be able to deliver the concept. how do you suggest to do so without the background?

Reply:
Abstract should be a high level human readable description of the specification.

Motivation should describe *why* this EIP is a good idea, but it doesn't need to be exhaustive.

Specification should only describe the specification itself, not its reasoning/motivation.

Rationale should describe why you made specific decisions within the specification (like why a particular value was chosen).

----

Something to keep in mind, EIPs are specifications, they are not meant to be exhaustive documents on a particular topic.  If you want to discuss a lot about the background, reasoning, history, etc. you should put that in a blog, website, docs site, etc. and leave the EIP just as the specification.